acceptance/python/Dockerfile

```diff
@@ -5,4 +5,4 @@ RUN pip install --upgrade \
 	pycco \
 	websocket-client~=0.47.0 \
 	pytest~=3.0 \
-	bigchaindb-driver==0.5.0a2
+	bigchaindb-driver==0.5.0a4
```

Comment:
I wonder if we should _always_ update this line as part of our [release process](https://github.com/bigchaindb/bigchaindb/blob/master/RELEASE_PROCESS.md)? We could add a sub-step to the release process (in step 1).

Reply:
Actually, maybe it's part of the Python driver release process.

Reply:
To me, this depends on the driver, should we put something there?

Reply:
If the driver had its own release process document, we could put a task in that, but it doesn't have one. However, there _is_ an open issue to create one. I'll add a comment to that issue, noting that the process should include updating this line in the bigchaindb/bigchaindb repo.